quantum/rgblight.h

```diff
@@ -170,6 +170,45 @@ enum RGBLIGHT_EFFECT_MODE {
 #        define RGBLIGHT_LIMIT_VAL 255
 #    endif
 
+// velocikey customization
```

Comment:
If anything, I'd say make these non-specific to velocikey, and make them just global rgblight settings.  (eg, change the names to reflect rgblight, rather than velocikey).

But that may be better as a separate PR.

Reply:
My reasoning was that, at least from my understanding of the code, these variables don't have any effect when not used with Velocikey. But maybe I got it wrong, and could of course rename it.

Reply:
Any updates on this?